doc/_themes/saltstack2/layout.html

```diff
@@ -273,7 +273,7 @@
                                  <!--
                                  <a href="https://saltstack.com/saltstack-enterprise/" target="_blank"><img class="nolightbox footer-banner center" src="{{ pathto('_static/images/enterprise_ad.jpg', 1) }}"/></a>
                                  -->
-                                 <a href="https://saltconf.com/ad" target="_blank"><img class="nolightbox footer-banner center" src="https://get.saltstack.com/rs/304-PHQ-615/images/Salt-docs-ad-330x330.png"/></a>
+                                 <a href="https://saltconf.com/ad" target="_blank"><img class="nolightbox footer-banner center" src="https://gitlab.com/saltstack/open/salt-branding-guide/-/raw/master/misc/banners/try_salt_now.png"/></a>
```

Comment:
The salt conf urls, ie, the image target links also lead nowhere related to salt conf. At least, yesterday they didn't....

Reply:
The links are owned by @janae417 and the team managing wordpress, and have it on their backlog to update the redirects. The links will eventually point to the correct location, so I'm leaving them as-as so they don't need to be modified within the repo.

Reply:
ok